spec.bs

```diff
@@ -937,9 +937,10 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
     Note: The JavaScript code calling {{Navigator/runAdAuction()}} is responsible for resolving
       |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"] only when the {{Promise}}
       returned from an associated {{WindowOrWorkerGlobalScope/fetch()}} that was made with the

```

Comment:
@caraitto Is it to link to `{{WindowOrWorkerGlobalScope/fetch()}}` here, or should I say something like "a [=request=] whose [=request/initiator type=] is `"fetch"`", like [what you had](https://github.com/WICG/turtledove/blob/main/spec.bs#L3136C11-L3136C71) in your PR?

Reply:
Dominic had me change to the request whose type is fetch language in my PR (I think because there may be another API entry point that gets that request type?), so maybe go with that?

Reply:
thanks, done.